### ISSUE TITLE
Uforklaret brug af 0..n

### ISSUE THREAD

**Issue by JanEmilLarsen** [digst/DCAT-AP-DK#32]
Multiplicitet/kardinalitets-angivelsen 0..n i dcat:Dataset (og andre steder):

Fx forekommer

dcat:spatialResolutionInMeters[0..n] og dcat:temporalResolution[0..n]

og i begge tilfælde er det uklart hvilke værdier af n>1 der kan give mening.

Hvad er betydningen?
Skulle der have stået [0.1] for at indikere at en værdi er optionel?

**Comment by aidig** [comment]
Mulitpliciterne er defineret således i den europæiske anvendelsesprofil, jf. https://joinup.ec.europa.eu/collection/semantic-interoperability-community-semic/solution/dcat-application-profile-data-portals-europe
Der er dog et åbent issue vedrørende dcat:temporalResolution på deres aktuelle issue tracker, se: https://github.com/SEMICeu/DCAT-AP/issues/114 Såfremt multipliteten ændres i den næste version af DCAT-AP, vil DCAT-AP-DK skulle rettes tilsvarende i en fremtidig revision.

**Comment by JanEmilLarsen** [comment]
Ja, jeg ser at det samme forhold gør sig gældende i den profil der henvises til.
Heller ikke der se jeg nogen forklaring eller eksempel på hvilken mening det kunne have.

Hvilken mening kan man fx have med dcat:spatialResolutionInMeters[1] = 1m og dcat:spatialResolutionInMeters[2] = 4m?

Der synes at være samme observation og formodning som min i det issue der henvises til.

Har I ingen (selvstændig) opfattelse af den påtænkte semantik, og er bindingen til DCAT-AP så stærk at I vælger at videreføre fejl?

**Comment by pebran** [comment]
> 
> 
> Multiplicitet/kardinalitets-angivelsen 0..n i dcat:Dataset (og andre steder):
> 
> Fx forekommer
> 
> dcat:spatialResolutionInMeters[0..n] og dcat:temporalResolution[0..n]
> 
> og i begge tilfælde er det uklart hvilke værdier af n>1 der kan give mening.
> 
> Hvad er betydningen?
> Skulle der have stået [0.1] for at indikere at en værdi er optionel?

o..n angiver at egenskaben ikke behøver at forekomme, men hvis den gør er der ingen øvre grænse for hvor mange gange den kan forekomme.

**Comment by JanEmilLarsen** [comment]
Ja, jeg er helt klar over at 0..n betyder at egenskaben kan forekomme 0 til n gange (og da n er uspecificeret, i samme betydning som *) uden øvre grænse. 

Spørgsmålet går på hvilken relevant semantik der kan være i at egenskaben kan forekomme mere end én gang.

**Comment by pebran** [comment]
> 
> 
> Ja, jeg ser at det samme forhold gør sig gældende i den profil der henvises til.
> Heller ikke der se jeg nogen forklaring eller eksempel på hvilken mening det kunne have.
> 
> Hvilken mening kan man fx have med dcat:spatialResolutionInMeters[1] = 1m og dcat:spatialResolutionInMeters[2] = 4m?
> 
> Der synes at være samme observation og formodning som min i det issue der henvises til.
> 
> Har I ingen (selvstændig) opfattelse af den påtænkte semantik, og er bindingen til DCAT-AP så stærk at I vælger at videreføre fejl?

Vi vælger ikke at videreføre fejl. Vi har videreført den multiplicitetsangivelse der blev anvendt i DCAT-AP.  Umiddelbart vil jeg give dig ret i at der ikke synes at være nogen mening i at have flere angivelser af værdier for dcat:spatialResolutionInMeters. Den problemstilling fangede vi ikke da vi lavede den danske AP.   
Du har altså muligvis fundet en fejl i DCAT-AP og det vil være meget fint og en stor hjælp hvis du kan bidrage til afklaring ved at stille spørgsmålet til DCAT-AP-gruppen.
